README.md

```diff
@@ -47,79 +47,143 @@ Several features are in beta testing and will be available in the next release s
 
 *If your want to join in our development team, please contact huaizhen001 @ e.ntu.edu.sg*
 
-## Installation
+## Demo
 
-### using pip
+The below figures illusrates the 
+| Web frontend |   Workflow     |
+|:------------:|:--------------:|
+| <img src="https://i.loli.net/2020/12/10/4FsfciXjtPO12BQ.png" alt="drawing" width="500"/> | <img src="https://i.loli.net/2020/12/10/8IaeW9mS2NjQEYB.png" alt="drawing" width="500"/>    |
+
+
+## Installation Guide
+
+### Prerequisites
+
+- A GNU/Linux environment(Ubuntu preferred)
+- [Docker](https://docs.docker.com/engine/install/)
+- [Docker Compose](https://docs.docker.com/compose/) (for Installation via Docker)
+- [TVM](https://github.com/apache/incubator-tvm) and `tvm` Python module
```

Comment:
I suppose TVM and TensorRT should be optional software. We do not require user to install it for enjoying modelci